### ISSUE TITLE
 Contributing in russian translation

### ISSUE THREAD

**Issue by farggus** [LibreTime/libretime#973]
Hi! I observed that last translation to russian was made about two years ago, how it is noted in zanata, and is marked that work is 100% done. But, how I can see, new text blocks and frases appears in UI, which can be translate. I'd like to contribute and to translate them

**Comment by Robbt** [comment]
I think that we never fully transitioned over to using Zanata, if you are familiar with it you can chime in on #78 and help us integrate that tool. Most contributions for translations have been done just by editing the .po files such as #895 - if you want to contribute new Russian translations that way we can merge your PR for any missing vocabulary. Thanks for your interest in helping and let us know if you need any more help in terms of how to contribute. 

As long as you are familiar with git etc then #895 should provide an example for how to translate by editing the text files you would just need to find the file for the Russian language.

**Comment by farggus** [comment]
I thinked that Zanata is main tool for translation Libretime and i not familiar with it unfortunately. I used other soft to edit .po file.
So, i've updated the russian translation and have added the new PR #985 with proposed changes. 
I had to make some changes to other .phtml and .php files to made translation fully working.

If i add  any new translation in future they will add to this PR automatically, or i need to open new PR?
